README.md

```diff
@@ -1,79 +1,142 @@
+- [ ] If you want the ability to manage your load-balancing configuration with the same tools you use to manage your Kubernetes cluster, you might need NKL.
 
->>**This is a replacement for a Cloud Providers "Service Type Loadbalancer", that is missing from On Premises Kubernetes Clusters.**
+### Why NKL?
 
-<br/>
+NKL provides a simple, easy-to-manage way to automate load balancing for your Kubernetes applications by leveraging NGINX Plus hosts running outside your cluster.
 
-## Overview
+NKL installs easily, has a small footprint, and is easy to configure and manage.
 
-- Create a new K8s Controller, that will monitor specified k8s Services, and then send API calls to an external Nginx Plus server to manage Nginx Upstream servers automatically.  
-- This will `synchronize` the K8s Service Endpoint list, with the Nginx LB server's Upstream server list.  
-- The primary use case is for tracking the NodePort IP:Port definitions for the Nginx Ingress Controller's `nginx-ingress Service`.  
-- With the Nginx Plus Server located external to the K8s cluster, this new controller LB function would provide an alternative TCP "Load Balancer Service" for On Premises K8s clusters, which do not have access to a Cloud providers "Service Type LoadBalancer".
-- Make the solution a native Kubernetes Component, running, configured and managed with standard K8s commands.
+? {{review for embetterment}}: NKL does not require any specific domain knowledge for configuration, though you will have to understand NGINX configuration to get the most out of this solution. There is thorough documentation available about these specifics in the `docs/` directory.
 
-<br/>
+### What does NKL do?
 
-## Reference Diagram
+tl;dr:
 
-<br/>
+_**NKL is a Kubernetes controller that monitors Services and Nodes in your cluster, and then sends API calls to an external NGINX Plus server to manage NGINX Plus Upstream servers automatically.**_
 
-![NGINX LB Server](docs/media/nginxlb-nklv2.png)
+That's all well and good, but what does that mean? Well, Kubernetes clusters require some tooling to handling routing traffic from the outside world (e.g.: the Internet, corporate network, etc.) to the cluster. 
+This is typically done with a load balancer. The load balancer is responsible for routing traffic to the appropriate Kubernetes worker node which then forwards the traffic to the appropriate Service / Pod.
 
-<br/>
+If you are using a hosted web solution -- Digital Ocean, AWS, Azure, etc. -- you can use the cloud provider's load balancer service. Those services will create a load balancer for you. 
+You can use the cloud provider's API to manage the load balancer, or you can use the cloud provider's web console.
 
-## Sample Screenshots of Runtime
+However, if you checked the first box above, you are running Kubernetes on-premise and will need to manage your own load balancer. This is where NKL comes in.
```

Comment:
I think this may need some more words "However, if you checked the first box in "Who needs NKL?" above, then..."

Since it's a bit far away from that for me to remember that there were checkboxes

Reply:
Yep, that was kind of cheeky and presumptuous.